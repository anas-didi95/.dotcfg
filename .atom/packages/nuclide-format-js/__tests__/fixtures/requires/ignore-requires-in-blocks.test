const c = require('c');
const a = require('a');
const d = require('d');

if (true) {
  const b = require('b');
}

a();
c();
