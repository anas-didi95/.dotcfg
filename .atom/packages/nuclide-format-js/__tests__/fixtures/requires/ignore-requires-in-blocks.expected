const a = require('a');
const c = require('c');

if (true) {
  const b = require('b');
}

a();
c();
